src/probnum/filtsmooth/statespace/integrator.py

```diff
@@ -24,6 +24,16 @@ class Integrator:
     def __init__(self, ordint, spatialdim):
         self.ordint = ordint
         self.spatialdim = spatialdim
+        self.precon = None  # to be set later
+
+    def setup_precon(self):
+        # This can not be done in the init, due to the order in which
+        # IBM, IOUP, and Matern initialise their superclasses.
+        # Integrator needs to be initialised before LTISDE,
+        # in which case via LTISDE, self.precon is set to None
+        # To make up for this, we call this function here explicitly
+        # after the calls to the superclasses' init.
```

Comment:
this.

Reply:
I'm not sure I fully get why this is necessary. But I trust that it is so this is not a reason not to merge, but let's maybe talk about it briefly some day :)

Reply:
Short summary: both `integrator` and `SDE` have a `precon` attribute. `SDE`s set it to `None` as a default, but `Integrators` enjoy `NordsieckLikeCoordinates`. Therefore the order of inits matters. 
Unfortunately, due to the values in `driftmat`, etc., `Integrator.__init__` needs to be called before `SDE.__init__`, which messes up the `precon` field. This is what the fix is for.

Please ask me in person if this is not clear yet, I understand that it is a bit twisted...